CRITICAL - 2024-01-01 03:17:11 --> Undefined variable: menu
in APPPATH\Views\v_template.php on line 122.
 1 APPPATH\Views\v_template.php(122): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: menu', 'C:\\xampp\\htdocs\\MartDBU\\app\\Views\\v_template.php', 122, [...])
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\MartDBU\\app\\Views\\v_template.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('v_template', [], true)
 5 APPPATH\Controllers\Admin.php(14): view('v_template', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Admin->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-01-01 03:47:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-01-01 03:47:18 --> Call to undefined function form_open()
in APPPATH\Views\v_template.php on line 59.
 1 SYSTEMPATH\View\View.php(228): include()
 2 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('v_template', [], true)
 4 APPPATH\Controllers\Admin.php(14): view('v_template', [...])
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Admin->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-01-01 06:01:29 --> Undefined variable: jmlh_produk
in APPPATH\Views\v_admin.php on line 5.
 1 APPPATH\Views\v_admin.php(5): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined variable: jmlh_produk', 'C:\\xampp\\htdocs\\MartDBU\\app\\Views\\v_admin.php', 5, [...])
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\MartDBU\\app\\Views\\v_admin.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('v_admin', [], true)
 5 APPPATH\Views\v_template.php(156): view('v_admin')
 6 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\MartDBU\\app\\Views\\v_template.php')
 7 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('v_template', [], true)
 9 APPPATH\Controllers\Admin.php(18): view('v_template', [...])
10 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Admin->index()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 06:53:08 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 06:53:08 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 06:56:00 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 06:56:00 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 06:56:04 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 06:56:04 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 06:56:24 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 06:56:24 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 06:57:51 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 06:57:51 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 06:57:53 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 06:57:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 06:57:57 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 06:57:57 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 06:58:00 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 06:58:00 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 06:58:06 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 06:58:06 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\Modelsatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 07:01:34 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(12): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(12): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 07:01:34 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(12): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 07:07:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 07:07:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 07:07:21 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 07:07:21 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-01-01 07:07:22 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'db_martdbu', 3306, '', 0)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#8 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in C:\xampp\htdocs\MartDBU\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\MartDBU\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\MartDBU\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\MartDBU\app\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\MartDBU\app\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
#5 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(942): App\Controllers\Satuan->index()
#6 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
#7 C:\xampp\htdocs\MartDBU\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\MartDBU\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-01-01 07:07:22 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tbl_satuan`', [], false)
 3 APPPATH\Models\Modelsatuan.php(11): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Satuan.php(23): App\Models\ModelSatuan->AllData()
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Satuan->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Satuan))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
